SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1275,6 +1279,25 @@ TWeakObjectPtr<USpatialActorChannel> USpatialReceiver::PopPendingActorRequest(Wo
 	return Channel;
 }
 
+AActor* USpatialReceiver::FindSingletonActor(UClass* SingletonClass)
```

Comment:
FindSingletonActor seems to be generic enough to not be in USpatialReceiver. What if we move it to NetDriver instead?

Reply:
It's not a method you'd want to encourage to use, `GetAllActorsOfClass` is expensive, and only used here as a last resort.

Reply:
I agree, my motivation is that it is more of a utility function and not a part of SpatialReceiver. I'm fine with it staying in SpatialReceiver though.